code/modules/surgery/bodyparts/head.dm

```diff
@@ -259,7 +259,7 @@
 				eye_right.color = eyes.eye_color_right
 			. += eye_left
 			. += eye_right
-		else
+		else if(IS_ORGANIC_LIMB(src))
 			. += image('icons/mob/species/human/human_face.dmi', "eyes_missing", -BODY_LAYER, SOUTH)
```

Comment:
dont put another one of these checks, just nest it inside the other one

Reply:
I see. I have moved it now.